apps/emqx_bridge_rabbitmq/test/emqx_bridge_rabbitmq_v2_SUITE.erl

```diff
@@ -261,3 +300,31 @@ send_test_message_to_rabbitmq(Config) ->
         }
     ),
     ok.
+
+probe_bridge_api(Name, PathStr, Config) ->
```

Comment:
Nit: there's already `emqx_bridge_v2_testlib:probe_bridge_api/4`.

Reply:
Thanks I will change next time. 
make all probe_bridge_api to emqx_bridge_v2_testlib